src/sprout/views/dbtools/file_backend_migrate.php

```diff
@@ -0,0 +1,94 @@
+<?php
+use Sprout\Helpers\Enc;
+use Sprout\Helpers\Form;
+use Sprout\Helpers\Text;
+use Sprout\Helpers\Url;
+
+?>
+
+<style>
+.dbtools-box { color: #333; text-decoration: none; min-height: 150px; }
+.dbtools-box:hover { color: #333; text-decoration: none; box-shadow: 0 0 2px 2px #c3c7d4; }
+.dbtools-box h4 { margin: 0 0 0.5em 0; font-size: 20px; }
+.dbtools-box span { font-size: small; color: #666; }
+</style>
+
+<form action="<?php echo Url::current(); ?>">
+
+<input type="hidden" name="go" value="1" />
+
+<h3>There are approximately <?php echo (int) $files_count; ?> files to be migrated.</h3>
+
+<div class="white-box">
+    <h4>Please choose your action carefully:</h4>
+
+    <h5>Prepare:</h5>
+    <p>This will create copies on the chosen new backend,
+        and add migration type & date to te database.
+        <br>This will not change the current backend for the file.
+        <br>Use this to prepare large file systems before making the final config change.
+        <br>You will be required to run an 'Update database' action when the active backend config is changed.
+    </p>
+
+    <h5>Update:</h5>
+    <p>This will update the active backend in the record for any prepared migrations.
+        <br>This update the 'backend_type' field in the database, copied from 'backend_migrated', and will not create any new files.
+        <br>Updates will only be performed where backend_migrated matches the chosen new backend, and the date_migrated field is not empty.
+        <br>Use this to finalise a prepared migration.
+    </p>
+
+    <h5>Full:</h5>
+    <p>This will perform both of the above actions.
+        <br>Updates will only be performed where backend_migrated matches the chosen new backend, and the date_migrated field is not empty.
+        <br>Use this for fast migration of small volume file systems.
+    </p>
```

Comment:
I love this 2-stage migration approach.

Only I'm never reassured that the old files are not deleted. Perhaps (and this is more work) there's a 3rd stage that cleans up the old files.

"Prepare" also doesn't sit right. Maybe just "Copy" is enough. It was a bit weird to read that in the worker and see it do copy actions, which didn't seem like preparation steps at first.

Reply:
Prepare would be clearer. I'd previously just been doing a prepare run, making sure it worked, then doing a full run.
Mostly I have been leaving the originals there because fear :D